### PR TITLE
Rename flow exporters

### PR DESCRIPTION
**Pull request by jcreixell**
#### PR Description

  - Renames flow `integrations` to `exporters`. This is done to avoid confusion with Grafana Cloud integrations, which include metrics, logs, dashboards and alerts among other things (exporters are just a small piece of the integrations puzzle). This is also consistent with prometheus conventions.
  - Additionally, renames `node_exporter` to `node` for consistency with the other exporters.

#### Notes to the Reviewer

Is there anything else that we need to change outside this repo?

#### PR Checklist

- [x] CHANGELOG updated
- [x] Documentation added

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -44,11 +44,11 @@ Main (unreleased)
...
-  - `prometheus.integration.apache` collects metrics from an apache web server
+  - `prometheus.exporter.apache` collects metrics from an apache web server
     (@captncraig)
-  - `prometheus.integration.consul` collects metrics from a consul installation
+  - `prometheus.exporter.consul` collects metrics from a consul installation
     (@captncraig)
-  - `prometheus.integration.github` collects metrics from GitHub (@jcreixell)
+  - `prometheus.exporter.github` collects metrics from GitHub (@jcreixell)
 
 ### Enhancements
```
Comment (rfratto) [inline CHANGELOG.md:53]: Renaming these components is a breaking change, can you note the renamed components in the breaking changes section on line 13?

(It's outside of the changed hunk so I can't comment there directly)

File: docs/sources/flow/reference/components/prometheus.exporter.apache.md [hunk 2]
```diff
@@ -6,17 +6,17 @@
...
-The `prometheus.integration.apache` component embeds
+# prometheus.exporter.apache
+The `prometheus.exporter.apache` component embeds
 [apache_exporter](https://github.com/Lusitaniae/apache_exporter) for collecting mod_status statistics from an apache server.
 
 ## Usage
 
 ```river
-prometheus.integration.apache "LABEL"{
+prometheus.exporter.apache "LABEL"{
```
Comment (rfratto) [inline docs/sources/flow/reference/components/prometheus.exporter.apache.md:19]:
```suggestion
prometheus.exporter.apache "LABEL" {
```

**Review by rfratto — APPROVED** [review]
LGTM, but two things:

1. There's still some references to `prometheus.integration` components elsewhere in the docs, can you update those? 
2. There's some history with whether `node_exporter` should be called `node` or something else (or keep being node_exporter): #1231. 

I'll leave the final decision for what the name should be up to you, though, feel free to follow your gut here.

File: CHANGELOG.md (hunk 1 above)
Comment (marctc) [reply CHANGELOG.md:53]: Is it also a breaking change even if the changes were not part of a release?

Comment (jcreixell) [reply CHANGELOG.md:53]: I guess it is a breaking change only for the already released integrations, I will document those

**Comment by gaantunes** [comment]
I am onboard with this change, we might even want to change it on the integrations console in the future as well.

I am just not a 100% about changing `node_exporter` to `node`, since the first is very know as being related to unix-like monitoring in the community, and changing it to `node` might cause some confusion.

**Review by rfratto — APPROVED** [review]
:shipit:

**Pull request merged**